src/components/common/ValidateForm/ValidateForm.tsx

```diff
@@ -104,6 +105,8 @@ function ValidateFailure({ formId, validationError, onClickValidate, validatedTi
 export function ValidateForm({ id, onClickValidate, validationError, isChartRepo = false, validationStatus = "", configName, warning = "" }) {
     return (
         <div className="mt-16">
+            {!id && validationStatus != VALIDATION_STATUS.LOADER && 
+                <div className='infobar flex left bcb-1 eb-2 bw-1 br-4 mb-20 pt-10 pb-10 pr-16 pl-16'><img src={infoIcon} alt="add-worflow" className="icon-dim-18 mr-8" />Only public chart repositories can be connected as of now.</div>}
```

Comment:
Import SVG as ReactComponent & use it instead of passing path to img tag

Reply:
done